doorman/plugins/alerters/pagerduty.py

```diff
@@ -32,7 +32,9 @@ def handle_alert(self, node, match):
 
         details = {
             'node': node,
```

Comment:
Can't comment on lines not modified, but the `client_url` specified above would be ripe for passing a format string, to allow linking to the node id directly.